EmployeeTaskSolicy/Dto/EmployeeDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EmployeeTaskSolicy.Dto
+{
+    public class EmployeeDto
+    {
+        public int? Id { get; set; }
```

Comment:
l think only id should be nullable

Reply:
Better to have separate DTO's for input and output of data. So The DTO that you will return will be with ID and the one which you are receiving from Frontend will be without. And they can inherit from each other to avoid code duplication. About nullable` Id should never be Nullable because it represents bad design